python/elided_label/elided_label.py

```diff
@@ -69,6 +102,10 @@ def setText(self, text):
 
         :param text:    The text to set on the label
         """
+        # clear out the ideal width so that the widget can recalculate based on
+        # the new text
+        if hasattr(self, '_ideal_width'):
```

Comment:
Minor: would it be cleaner for it to always exist but contain None when it's unpopulated?